examples/atari/a3c/train_a3c.py

```diff
@@ -0,0 +1,177 @@
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+from __future__ import absolute_import
+from builtins import *  # NOQA
+from future import standard_library
+standard_library.install_aliases()  # NOQA
+import argparse
+import os
+
+# Prevent numpy from using multiple threads
+os.environ['OMP_NUM_THREADS'] = '1'  # NOQA
+
+import chainer
+import gym
+import gym.wrappers
+import numpy as np
+
+import chainerrl
+from chainerrl.agents import a3c
+from chainerrl import experiments
+from chainerrl import links
+from chainerrl import misc
+from chainerrl.optimizers.nonbias_weight_decay import NonbiasWeightDecay
+from chainerrl.optimizers import rmsprop_async
+from chainerrl import policy
+from chainerrl import v_function
+
+from chainerrl.wrappers import atari_wrappers
+
+
+class A3CFF(chainer.ChainList, a3c.A3CModel):
+
+    def __init__(self, n_actions):
+        self.head = links.NIPSDQNHead()
+        self.pi = policy.FCSoftmaxPolicy(
+            self.head.n_output_channels, n_actions)
+        self.v = v_function.FCVFunction(self.head.n_output_channels)
+        super().__init__(self.head, self.pi, self.v)
+
+    def pi_and_v(self, state):
+        out = self.head(state)
+        return self.pi(out), self.v(out)
+
+
+def main():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--processes', type=int, default=16)
+    parser.add_argument('--env', type=str, default='BreakoutNoFrameskip-v4')
+    parser.add_argument('--seed', type=int, default=0,
+                        help='Random seed [0, 2 ** 31)')
+    parser.add_argument('--outdir', type=str, default='results',
+                        help='Directory path to save output files.'
+                             ' If it does not exist, it will be created.')
+    parser.add_argument('--t-max', type=int, default=5)
+    parser.add_argument('--beta', type=float, default=1e-2)
+    parser.add_argument('--profile', action='store_true')
+    parser.add_argument('--steps', type=int, default=8 * 10 ** 7)
```

Comment:
Source: Noisy nets paper - "The DQN and A3C agents were training for
200M and 320M frames, respectively".